docs/CONFIG-VARS.md

```diff
@@ -387,3 +388,101 @@ postgres_servers = {
+| waf_policy | A JSON file with all the WAF_Policy rules | map | null | The WAF policy has few required components see the details below. |
+
+The `app_gateway_config` variable can contain none, some, or all of the parameters listed below:
+For the details of all the parameters that can be specified see: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/application_gateway
+
+| Name | Description | Type | Required | Default | Notes |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| sku | The Name of the SKU to use for this Application Gateway. | string | false | "Standard_v2" | If WAF policy is enabled the default is `"WAF_v2"` |
+| port | The port which should be used for this Application Gateway. | string | false | "443" |  |
+| protocol | The Protocol which should be used. | string | false  | "Https" | Possible values are Http and Https.|
+| backend_host_name | Hostname for the Application Gateway | string | false | null |Set this variable when using custom DNS. Not setting this will use Azure Public DNS to set the FQDN for Application Gateway Public IP.|
+| backend_trusted_root_certificate | The Trusted Root Certificate to use. | list(map(string)) | true | null  | List of map containing: name, data, or key_vault_secret_id. `key_vault_secret_id` is required if `data` is not set. |
+| ssl_certificate |The associated SSL Certificate which should be used for this HTTP Listener. | list(map(string)) | true | null | List of map containing: name, data, password or key_vault_secret_id. `key_vault_secret_id` is required if `data` is not set.|
+| identity_ids | Specifies a list of User Assigned Managed Identity IDs to be assigned to this Application Gateway. | list | false | null | Required when specifying key_vault_secret_id |
+| backend_address_pool_fqdn | A list of FQDN's which should be part of the Backend Address Pool. | list | true | null | Add the FQDN of your Ingress-Nginx. Note: You can specify the anticipated FQDN of your ingress without affecting the Gateway creation. This can also be manually updated later in Portal|
+| probe | Custom health probes to be created for the Application Gateway. | map | false | "default-probe" ||
+
+The `waf_policy` variable is json file which can contain none, some, or all of the parameters listed below:
+For the details of all the parameters that can be specified see: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/web_application_firewall_policy
+| Name | Description | Type | Required | Default | Notes |
+| :--- | :--- | :--- | :--- | :--- | :--- |
+| custom_rules | Specifies one or more custom_rules blocks  | list(map) | false | | |
+| policy_settings | Specifies a policy_settings block | map | false |  |  |
+| managed_rules | A managed_rules blocks | list(map) | true  | | |
+
+Example WAF Policy:
+```
```

Comment:
Change the line to include json along with the 3 ` ticks

Reply:
I'll update this.

Reply:
Updated in the latest commit change